src/pip/_internal/commands/configuration.py

```diff
@@ -19,15 +19,16 @@
 
 
 class ConfigurationCommand(Command):
-    """Manage local and global configuration.
+    """
+    Manage local and global configuration.
```

Comment:
With [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) in mind, I personally prefer the previous version.

Reply:
I agree that the new change doesn't follow PEP 257, but I did it this way to maintain correct rendering.

Doing something like the previous option ends up causing the rendering to look like

<img width="795" alt="Screenshot 2020-04-18 at 9 00 11 AM" src="https://user-images.githubusercontent.com/13155849/79627119-42ba5d80-8153-11ea-947b-c98e028959ff.png">

Reply:
I believe that it is the correct rendering (the same goes for output of `help`), which makes the summary stands out.

Reply:
I think both options are good, it's just a matter of personal preference, although I found that [download](https://github.com/pypa/pip/blob/master/src/pip/_internal/commands/download.py#L21) , [install](https://github.com/pypa/pip/blob/master/src/pip/_internal/commands/install.py#L72) and [show](https://github.com/pypa/pip/blob/master/src/pip/_internal/commands/show.py#L22)  do the same thing as the change I did. 

I will also let other chime in with their preferences, this should be trivial to change.

Reply:
Since no one else has chimed in with an opposition to the docstring format used above, and the PR has been approved, I am going ahead and resolving this conversation.